demo/src/BlankEditor.js

```diff
@@ -34,6 +34,7 @@ class BlankEditor extends Component<Props> {
             innerRef={(editor) => {
               this.editor = editor;
             }}
+            UPLOADCARE_API_KEY="demopublickey"
```

Comment:
We shouldn't be setting this– it's going to try to use the string `demopublickey` for the uploadcare widget.